components/sidebar.js

```diff
@@ -71,24 +78,46 @@ const Logout = styled.button`
   }
 `;
 
+const LoadingImage = styled.img`
+  height: 50px;
+  width: 50px;
+  padding-left: 10px;
+`;
+
 export default ({ hackathons, currentPath }) => {
+  const [loading, setLoading] = useState(false);
+
+  useEffect(() => {
+    setLoading(false);
+  }, [window.location.pathname]);
   return (
     <SidebarContainer>
-      <Header>nwPlus CMS</Header>
+      <HeaderContainer>
+        <Header>nwPlus CMS</Header>
+        {loading && <LoadingImage src={LoadingGif} />}
+      </HeaderContainer>
       <ItemContainer>
         <Website />
         <Item>Websites</Item>
       </ItemContainer>
       {hackathons.map((id) => {
         return (
-          <Link key={id} href={`/${id}/intro`} selected={currentPath === id}>
-            {id}
-          </Link>
+          <NextLink key={id} href="/[id]/intro" as={`/${id}/intro`}>
```

Comment:
isn't showing the `pointer: cursor` in the PR preview, might have to manually style this

Reply:
Should be fixed now